### ISSUE TITLE
Handling editor should be visible to authors

### ISSUE THREAD

**Issue by smilius42** [CCSDForge/episciences#350]
Authors of a submission should be able to see who is the editor handling their submission, and they should be able to send their handling editor a message. Concretely, this could be realized by showing the handling editor in the table shown on the 'My submissions' page (with clickable name to send a message to her/him) and by showing an 'Editor' box on the page of the submission (https://lmcs.episciences.org/paper/view?id=<paper-id>). This box should contain the name of the editor and a button 'Contact this editor' for sending a message to her/him. (Such a box and button already exist for administrators and chief editors).

**Comment by clason** [comment]
Note that not every journal wants this; some distinguish a handling editor (which is anonymous) for communicating editors (which are not, but are not involved in the decision and so are safe from author recriminations).

**Comment by laurentromary** [comment]
I concur with Stefan. We often have contributors to special issues wrongly communicating with us, editors in chief rather than their handling editors.

**Comment by smilius42** [comment]
As far as I am aware there is no difference between handling and communicating editors in Episciences. There are only editors. What I mean is that authors should be able to see and communicate with editor(s) who is/are assigned to their submission in the system. 

It's exactly that centralized way of going through the EiC that we'd like to avoid. LMCS wants to allow authors to directly talk to their editors, and the system should support it. The EiC can still be involved in cases where one or the other side does not react to communication.

**Comment by rtournoy** [comment]
> Note that not every journal wants this; some distinguish a handling editor (which is anonymous) for communicating editors (which are not, but are not involved in the decision and so are safe from author recriminations).
> 
> A possibility of (anonymously) contacting the handling editor from the submission page would be convenient, but the normal approach is sending an email to the contact address, which then (possibly via an editorial secretary) gets forwarded to the Editor in Chief -- who is usually in a much better position to poke the handling editor into action anyway...

Thanks for your feedback, we can accommodate the difference and make things optional.

**Comment by clason** [comment]
Yes, my whole point is that not _everyone_ will want this information to be public, so it should be made optional, and independent of the possibility of contacting handling editors directly (which may also not be wanted in every case?)

**Comment by rtournoy** [comment]
We are going to evaluate the cost of this as an option

**Comment by clason** [comment]
In any case, a "contact editor" button on the submitter's article management page -- whether that name is show or not -- would be very useful!

**Comment by rtournoy** [comment]
Please have look at the feature we have added today - disabled by default. You can have a preview with the documentation:
https://doc.episciences.org/authoreditorcommunication/

For LMCS there's no sandbox but we can create one to test, or give you access to a shared one to try

I will close the issue, but feel free to comment, and reopen if needed, or reach out through the support email
